exporter/jaeger/jaegergrpcexporter/exporter.go

```diff
@@ -74,7 +77,7 @@ func (s *protoGRPCSender) pushTraceData(
 	}
 
 	_, err = s.client.PostSpans(
-		context.Background(),
+		metadata.NewOutgoingContext(ctx, s.metadata),
```

Comment:
on the big scheme of things this may not make a noticeable difference but at first it seems reasonable to only update the context if `len(s.metadata) > 0`, something like the ocagent does at https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/blob/b3d61dbd6252d1ec58f1cd6cebfa6bc672f1e816/ocagent.go#L222

Reply:
done